client/data/media/use-google-photos-picker-media-items-query.ts

```diff
@@ -0,0 +1,52 @@
+import { useQuery } from '@tanstack/react-query';
+import wp from 'calypso/lib/wp';
+
+interface Thumbnails {
+	large: string;
+	medium: string;
+	thumbnail: string;
+}
+
+interface Media {
+	ID: string;
+	name: string;
+	date: string;
+	file: string;
+	URL: string;
+	type: string;
+	mime_type: string;
+	width: number;
+	height: number;
+	size: number;
+	extension: string;
+	guid: string;
+	thumbnails:
+		| Thumbnails
+		| {
+				fmt_hd: string;
+				fmt_dvd: string;
+				fmt_sd: string;
+		  };
+}
+
+interface ResponseData {
+	found: number;
+	media: Media[];
+	meta?: {
+		next_page: boolean;
```

Comment:
There's also an `account` under meta, which contains two keys:

```
account:{ 
image:string
name:string
}
```

Reply:
I just found it's with dash: `post-thumbnail`